src/main.rs

```diff
@@ -30,9 +41,22 @@ fn main() -> std::io::Result<()> {
         cmd.args(json);
 
         cmd.args(run.to_hyperfine_params());
-        println!("Running: {cmd:?}");
-        cmd.output()?; // TODO: Catch possible errors
-        files_to_be_merged.push(output);
+        info!("Running: {cmd:?}");
+
+        let result = cmd.output()?;
+        if result.status.success() {
+            debug!("Benchmark run successful");
+            files_to_be_merged.push(output);
+        } else {
+            let msg = result.stderr;
+            let s = util::to_string(msg);
+            warn!("Run[{label:?}] failed with '{s:?}'");
+            if s.contains("The setup command terminated with a non-zero exit") {
+                eprintln!("[Warning] Run {label:?} failed. Setup failed");
+            }
+            eprintln!("[Warning] Run {label:?} failed. Skipping... -> Hyperfine: {s:?}");
+            eprintln!("[Warning] {cmd:?}");
+        }
```

Comment:
Handle errors in the outcome of the command by checking its `status` property. Could this be improved?

Reply:
Fixed in 7b2da22